src/devices/SkiaSharpAdapter/SkiaSharpImageFactory.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SkiaSharp;
+
+namespace Iot.Device.Graphics.SkiaSharpAdapter
+{
+    /// <summary>
+    /// This image factory uses SkiaSharp as backend
+    /// </summary>
+    public class SkiaSharpImageFactory : IImageFactory
+    {
+        /// <inheritdoc />
+        public BitmapImage CreateBitmap(int width, int height, PixelFormat pixelFormat)
+        {
+            return new SkiaSharpBitmap(width, height, pixelFormat);
+        }
+
+        /// <inheritdoc />
+        public BitmapImage CreateFromStream(Stream file)
+        {
+            using var image = SKImage.FromEncodedData(file);
```

Comment:
once this gets disposed presumably bitmap won't?

Reply:
Jup, it wouldn't work otherwise...